assets/sass/components/_variables.scss

```diff
@@ -1,6 +1,18 @@
 /* =Variables
 -------------------------------------------------------------- */
-$icon-color: #ff0000;
-$primary-color: #ff0000;
+$icon-color: #F0648B;
```

Comment:
Can you update the variables name following this guidelines:  http://islovely.co/posts/how-to-name-color-variables-in-sass/

Thank you.